fiftyone/core/service.py

```diff
@@ -241,6 +241,32 @@ def stop(self):
         self.child = None
 
 
+class ExternalDatabaseService(Service):
+    """Service that manages the external MongoDB database."""
+
+    service_name = "ext-db"
+    allow_headless = True
+
+    @property
+    def command(self):
+        return ["sleep", "14d"]
```

Comment:
This command fails on Mac. Windows will have to be supported too. It might be worthwhile to tweak https://github.com/voxel51/fiftyone/blob/develop/fiftyone/service/main.py to support command-less services that just has a cleanup call.

It seems a bit rude to start unnecessary processes on a users computer...

Reply:
Good to know. I can definitely look into making a command-less service.

Reply:
I modified service to run without a command, so there is no unnecessary sub-process and behavior should be consistent across each OS.